my-ansible-playbook.yml

```diff
@@ -3,26 +3,11 @@
   remote_user: ec2-user
 
   vars_prompt: 
-    - name: secret_key
-      prompt: What is the secret key
+    - name: connection_string

```

Comment:
Nice work keeping this updated, but don't feel _obliged_ to keep Ansible up to date going forwards as we won't be using it again